datastore/src/main/java/com/ds/datastore/Utilities.java

```diff
@@ -22,12 +22,12 @@
     Logger logger = LoggerFactory.getLogger(Utilities.class);
 
     @Retry(name = "retry")
-    public HttpResponse<String> createConnection(String address, JsonObject jso, String serverAddress, Long id, String requestType) throws Exception{
+    public HttpResponse<String> createConnection(String address, JsonObject jso, String serverAddress, Long id, String requestType, String orderID) throws Exception{
         Gson gson = new Gson();
         String json = gson.toJson(jso);
         Builder builder = HttpRequest.newBuilder()
             .uri(new URI(address))
-            .headers("Content-Type", "application/json;charset=UTF-8");
+            .headers("Content-Type", "application/json;charset=UTF-8", "Content-Type", "application/octet-stream;charset=UTF-8");
```

Comment:
Im unlcear about the need for this change?

Reply:
It solved this error for me. However, for some reason, the error still appears for Tuvia.
<img width="649" alt="Screenshot 2021-07-27 111506" src="https://user-images.githubusercontent.com/54090340/127180191-2a65ac26-6419-432b-95b7-e08d42ffc2b5.png">

Reply:
Actually, I just got that error again now even with this new code. Any suggestions on how to solve this?

Reply:
when are you getting that error?

Reply:
![image](https://user-images.githubusercontent.com/54090340/127182315-e1e225bb-d569-4b60-9262-08b994674880.png)